app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,5 @@
+class QuestionsController < ApplicationController
+	def index
+		@questions = Question.paginate(page: params[:page])
```

Comment:
please configure your IDE to use 2 spaces for indentation
and fix indentations here